packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '../../components/library'
```

Comment:
Same here, please let us know if anything has changed. It's most probably just bringing this logic to a new component with copy/pasting, yet the fact that I can't properly check what has changed may introduce bugs that go unnoticed until we're in prob.